pyiron/sphinx/base.py

```diff
@@ -95,7 +126,16 @@ def id_spx_to_pyi(self):
 
     @property
     def plane_wave_cutoff(self):
-        return self.input["EnCut"]
+        if "eCut" in self.input.basis:
+            if "EnCut" in self.input.basis["eCut"]:
+                return self.input["EnCut"]
+            else:
+                warnings.warn("WARNING: cutoff was modified "
+                + "directly in job.input.basis; the following "
+                + "value is in Ry)")
+                return self.input.basis["eCut"]
```

Comment:
The pyiron units should always be SI-Units. https://pyiron.github.io/source/faq.html#what-units-does-pyiron-use